src/apps/dashboard/dashboard.dev.tsx

```diff
@@ -156,10 +156,8 @@ export default {
       defaultValue: "Borrow before @date",
       control: { type: "text" }
     },
-    // Config
-    thresholdConfig: {
-      defaultValue:
-        '{\n      "colorThresholds":{\n      "danger":"0",\n      "warning":"6"\n   }\n   }',
```

Comment:
[Some places it seems as if we move from primitive values to JSON objects even for single keys](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/633/files#diff-0c0b0afee3b4a39016462d76c44101e19d81c3e17506f2e629e366c88cfc7b3fL31).

This takes a JSON object which now only has a single key and reduces it to a primitive value. 

Do you have any thoughts about in which situations one should use one approach over the other?

Reply:
I agree. It is inconsistent. In the future we should always handle config as JSON objects. In that way we can deal with the types as boolean and number natively.
I'lla add a comment about it.